deployment/deployment-grpc-v2/02-contour.yaml

```diff
@@ -36,6 +36,11 @@ spec:
         volumeMounts:
         - name: contour-config
           mountPath: /config
+        readinessProbe:
```

Comment:
Should this be a readinessProbe or a liveness probe? I'm guessing liveness as this isn't about "should this thing get traffic" but rather "is this thing working or should we restart it"?

Reply:
This is straight out of @mattmoyer 'a docs. I didn't add /live and /ready because the former felt redundant.

Reply:
The library will serve both `/live` and `/ready` so we might want to wire up both. The status they return can be different depending on which checks we register, but this doesn't register any checks so far.

Reply:
If we are don't have any checks then this is a simple "is the server alive" check and liveness is probably the way to go.